src/main/java/frc/robot/RobotContainer.java

```diff
@@ -51,6 +51,16 @@ public class RobotContainer {
           () -> MathUtil.applyDeadband(driverXbox.getLeftX(), OperatorConstants.LEFT_X_DEADBAND),
           () -> driverXbox.getRawAxis(2));
 
+  Command driveFieldOrientedAnglularVelocityPrecision =
```

Comment:
this is find but would it be better to have curves on the joysticks for madison rather than being a binary toggle? right now it's purely linear, we did curves last year so you can take a look at that

Reply:
see this: https://github.com/team614frc/Team-614-Crescendo/blob/main/src/main/java/frc/robot/RobotContainer.java#L191-L201

Reply:
This is what Madison would like to go with for now, I’ll explore curves later on.

Reply:
<img width="1281" alt="Screenshot 2024-12-05 at 3 50 01 PM" src="https://github.com/user-attachments/assets/20215d67-2cf2-43eb-aec0-ef17045890d7">

there's a couple different options to play around with if you have time

Reply:
I’ll look into it, but I don’t have time to right now, just left robotics and Madison and Kerns said it was good to go

thanks Minh 👍